projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -60,11 +60,10 @@ class Initializer {
 		'jetpack-search',
 	);
 
-	const MY_JETPACK_SITE_INFO_TRANSIENT_KEY = 'my-jetpack-site-info';
-
+	const MY_JETPACK_SITE_INFO_TRANSIENT_KEY             = 'my-jetpack-site-info';
 	const UPDATE_HISTORICALLY_ACTIVE_JETPACK_MODULES_KEY = 'update-historically-active-jetpack-modules';
-
-	const MISSING_CONNECTION_NOTIFICATION_KEY = 'missing-connection';
+	const MISSING_CONNECTION_NOTIFICATION_KEY            = 'missing-connection';
+	const VIDEOPRESS_STATS_KEY                           = 'videopress-stats';
```

Comment:
Nitpick - might be a good idea to add `my-jetpack-` to the front of this transient key just in case the videopress plugin decides to use a similar transient name at some point in the future.

Reply:
Good idea, I'll add that